java/org/contikios/cooja/util/LogbackColor.java

```diff
@@ -0,0 +1,39 @@
+package org.contikios.cooja.util;
```

Comment:
This file is missing a copyright header.

Reply:
Fixed.